app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -196,7 +197,7 @@ private void showServerError(final ManualAddSearchEngineSettingsFragment that) {
     }
 
     @WorkerThread // makes network request.
-    private static boolean isValidSearchQueryURL(final String query) {
+    @VisibleForTesting static boolean isValidSearchQueryURL(final String query) {
```

Comment:
\[PMD\] INFO: The method 'isValidSearchQueryURL' has a Cyclomatic Complexity of 5\.

Reply:
@sputnikci It wasn't me! 😝

Reply:
Do you have a high level overview for what this warning means and how we should generally address it? I have the wikipedia page open for Cyclomatic Complexity and it looks non-trivial.